### ISSUE TITLE
TRP and GW2 UI

### ISSUE THREAD

**Issue by TheTark** [Total-RP/Total-RP-3#661]
## Describe the issue
Anytime I have the UI on TRP will not allow things to work, aka, abilities, items, prompts, potions all of it.

## Expected behaviour
I expect them to have no interference with eachother so I may keep TRP 3 on while I use my abilities, or RP and use items

## Issue checklist
Please select any applicable options from the below list.

> You can replace the "[ ]" with "[x]" for each option below to tick it.

- [ X] The issue occurs with the [latest version] of the addon.
- [ ] The issue occurs with Total RP 3 as the only enabled addon.
- [X ] The issue occurs only with other addon(s) enabled (please provide a list).

## Steps to reproduce
Simply logging in brings up the prompt to disable trp3

## Additional files
Please attach any additional applicable files such as screenshots, [error logs], or [profile data files]. Files can be dragged and dropped into this report to attach them for upload.

---

[error logs]: https://www.curseforge.com/wow/addons/bugsack
[profile data files]: https://github.com/Total-RP/Total-RP-3/wiki/Saved-Variables
[latest version]: https://www.curseforge.com/wow/addons/total-rp-3/files

**Comment by Solanya** [comment]
From what we can tell, the GW2 UI does a lot of things that are known to lead to taint issues like the ones you're encountering.

At best you can provide a log by using `/console taintlog 2`, reload, play until you encounter the issues and send the `taint.log` file in `World of Warcraft\_retail_\Logs` so we can look at the specifics of the issues you're encountering.

**Comment by TheTark** [comment]
[taint - Copy.log](https://github.com/Total-RP/Total-RP-3/files/9995560/taint.-.Copy.log)

**Comment by Meorawr** [comment]
I'm not seeing any blocked action errors in that taint log. The file is overridden if you `/reload` or log in on another character, so the steps you'd need to follow are:

1. Log in, run `/console taintlog 2`.
2. Run `/reload` - this is to ensure we generate log with all startup taint recorded.
3. Play the game normally until you get an "Interface action blocked by an addon" message in the chat _or_ as a popup.
4. Run `/console taintlog 0`.
5. Run `/reload`.
6. Attach the generated taint log here.

**Comment by TheTark** [comment]
I promise you I'm not making it up, it happens everytime I reload, use an item in my inventory, like the wow anniversary token, pots and some other items. even occurring sometimes when I use abilities 
[taint - Copy.log](https://github.com/Total-RP/Total-RP-3/files/9995613/taint.-.Copy.log)

**Comment by Solanya** [comment]
It seems the issue involves Extended, we're putting a workaround in place in the version for next week and hoping for the best.

**Comment by TheTark** [comment]
Yeah disabling Extended seems to have helped. Not sure why WoW kept making me just disable TRP3 for it, but it makes sense.

**Comment by Solanya** [comment]
The part triggering the issue is in the initialization functions of Extended which is technically called by TRP3 (with one of GW2 UI's libraries quickly spreading this to Edit Mode)

**Comment by TheTark** [comment]
yeah GW2 UI did take a bit to get launched for DF, and this Edit mode seems to cause more problems then it solves.

Thanks for helping out and getting back to me. and I look forward to all future additions to TRP3
